recipes/napari-figure/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "napari-figure" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-figure-{{ version }}.tar.gz
+  sha256: 3d9e379c432424249aac19d277e91e478721fbab5a65793975af759c7a8e3a7a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - magicgui
+    - microfilm
+    - napari
+    - numpy
+    - python >=3.8
+    - qtpy
```

```suggestion
    - qtpy
    - scikit-image
    - tifffile
    - vispy
    - matplotlib-base
```

A few ones missing in the metadata. e.g. https://github.com/BIOP/napari-figure/blob/v0.1.1/src/napari_figure/figure_widget.py#L13